README.md

```diff
@@ -4,6 +4,33 @@ SQL Runner allows a platform developer to run SQL against a backend,
 for data curation activities.
 Its use is governed by the [platform developer access policy](https://docs.opensafely.org/developer-access-policy/).
 
+## Handling Tier One Opt Outs
+
+To handle Tier One Opt Outs (T1OOs),
```

Comment:
```suggestion
To handle Type One Opt Outs (T1OOs),
```

Reply:
If the zeros weren't bad enough, now this! 😆